dashboards/tests/test_grafana_sync.py

```
@@ -54,41 +60,54 @@ def test_api_get_prints_error_on_non_200(monkeypatch, capsys):
 def test_resolve_folder_uid_exits_if_not_found(monkeypatch):
     monkeypatch.setenv("GRAFANA_URL", "https://test.grafana.net")
     monkeypatch.setenv("GRAFANA_TOKEN", "tok")
     monkeypatch.setenv("GRAFANA_FOLDER", "Missing")
     m = import_module()
-    with patch.object(m, "api_get", return_value=[{"uid": "d1", "folderTitle": "Other", "folderUid": "abc"}]):
+    with patch.object(
+        m,
+        "api_get",
+        return_value=[{"uid": "d1", "folderTitle": "Other", "folderUid": "abc"}],
+    ):
         with pytest.raises(SystemExit):
```

Ruff (0.15.9)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)